helios/models.py

```diff
@@ -1528,7 +1525,8 @@ def itervoters(self, email_validator=validate_email, preferred_encoding=None):
         if self.voter_file_content:
             voter_data = base64.b64decode(self.voter_file_content.encode())
         else:
-            voter_data = open(self.voter_file.path, "rb").read()
+            with open(self.voter_file.path, "rb") as voter_data:
+                voter_data.read()
```

Comment:
New code looks like it's not going to save the read content of the file into the variable, but I'm not 100% sure if that's the case. Just letting you know it looks suspicious.

Reply:
Yes, that's wrong. It should read:
```
with open(...) as f:
    voter_data = f.read()
```